sshd-core/src/test/java/org/apache/sshd/certificates/GenerateOpenSSHClientCertificateTest.java

```diff
@@ -154,7 +150,12 @@ public void signCertificate() throws Exception {
                 .publicKey(clientPublicKey)
                 .id("user01")
                 .principals(Collections.singletonList("user01"))
+                .criticalOptions(Arrays.asList(
+                        new OpenSshCertificate.CertificateOption("force-command", "/path/to/script.sh"),
+                        new OpenSshCertificate.CertificateOption("source-address", "127.0.0.1/32"),
+                        new OpenSshCertificate.CertificateOption("verify-required")))
                 .extensions(Arrays.asList(
+                        new OpenSshCertificate.CertificateOption("no-touch-required"),
```

Comment:
A bit off-topic, but I wonder: should non-sk keys even accept this option?

Reply:
hi @tomaswolf 
Based on specs I believe no-touch-required should be used only with sk keys. 
I added it here for completeness of all available critical options/extensions as in the code **CertificateOptions** are not constants so we do not have enumeration.